orchestra/migrations/0076_auto_20180621_1524.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
```

Comment:
We never change migrations across PRs. If the old one was committed to master, you should create a new one.

Reply:
@marcua Do you mean I should create a migration on top of the old one or delete the old migration and create a new one.

Btw for some reason, Github is showing the diff between two different files here. Instead of showing two different changes (1 file created and 1 file deleted).